data/pigui/views/game.lua

```diff
@@ -168,7 +168,7 @@ local function drawGameModules(delta_t)
 			local ok, err = ui.pcall(module.draw, module, delta_t)
 			if not ok then
 				module.disabled = true
-				logWarning(err)
+				-- TODO: visually notify the user of an error
```

Comment:
To be fixed prior to merge?

Reply:
No - this will be resolved by adding a queue for error notices and displaying a popup/notification to the user in-UI. Well beyond the scope of this change, which increases the amount of information present in an error log.